bgpd/bgpd.h

```diff
@@ -792,6 +802,19 @@ struct bgp_filter {
 		char *name;
 		struct route_map *map;
 	} usmap;
+
+	/* Advertise-map */
+	struct {
+		char *aname;
+		struct route_map *amap;
+
+		bool condition;
+
+		char *cname;
+		struct route_map *cmap;
+
+		enum advertise advertise;
```

Comment:
This would sound better with something like `update_type` or so, IMO.

Reply:
enum update_type { WITHDRAW, NLRI }; is this fine ? or 
enum update_type { WITHDRAW, ADVERTISE }; ?